actionpack/lib/action_dispatch/middleware/params_parser.rb

```diff
@@ -13,6 +19,7 @@ def initialize(message, original_exception)
       end
     end
 
+    #The default parsers list includes just JSON parser
```

Comment:
We should remove this; it's documenting something that's both obvious from the code, and not something we have any reason to commit to in our API.